codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -78,58 +134,112 @@ mitre->miter
 modelled->modeled
 modelling->modeling
 mould->mold
+moulds->molds
 nasalisation->nasalization
 neighbour->neighbor
 neighbours->neighbors
+normalisation->normalization
 normalise->normalize
 normalised->normalized
 normalises->normalizes
+normalising->normalizing
 optimisation->optimization
 optimisations->optimizations
+optimise->optimize
 optimised->optimized
 optimiser->optimizer
+optimises->optimizes
+optimising->optimizing
 organisation->organization
 organisations->organizations
 organise->organize
```

Comment:
Organiser

Reply:
Also added organisers, as in "The organisers of the event".